image/Dockerfile

```diff
@@ -13,7 +13,8 @@ RUN yum --enablerepo=extras install -y epel-release && \
 COPY tools /opt/intly/tools
 RUN chown -R 1001:root /opt/intly
 
-RUN find /opt/intly/tools -type f -exec chmod +x {} \;
+RUN find /opt/intly/tools -type f -exec chmod +x {} \; && \
+    mkdir /.kube && touch /.kube/config && chmod -R 777 /.kube
```

Comment:
does it need to be that permissive? does applying the above config `chown -R 1001:root` for the .kube folder fail?

Reply:
Yes, even just `744` isn't enough. I'll see if there is some other way to make it less permissive.

Reply:
what about 774? In theory just the group and the owner should need to write files in there

Reply:
774 works, updated.